scripts/make_toy_data.py

```diff
@@ -82,19 +90,13 @@
 args.add_argument(
     '--num-genes', default=80, type=int,
     help=(
-        'Number of Genes'
+        'Number of genes (rows)'
     )
 )
 args.add_argument(
     '--num-cells', default=None, type=int,
     help=(
-        'Number of cells'
-    )
-)
-args.add_argument(
```

Comment:
I appreciate the performance improvement and increased reliability of parsing a fetched copy of an Ensembl GTF but also advocate for keeping the pre-loaded genes option. 

Two possible uses for pre-loaded genes:
1. need to generate a matrix that has more genes than the most current human Ensemble annotation. (make_toy_data.py currently would limit the generated toy matrix to the number of genes in the fetched human Ensemble annotation).
2. need to simulate/replicate visualization with a specific list of gene names (eg. non-human gene names or an expression matrix for nonRNAseq data, scATAC-seq "gene names" would be genome coordinate ranges) for troubleshooting

Reply:
Helpful context, I've restored `--preloaded-genes` (5474edd).  I've also noted those use cases in the `--help` docs for that argument (0646433).